src/index.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import {BrowserRouter as Router } from 'react-router-dom';
 import App from './App';
+import Main from './components/Main';
+import { GlobalStyle } from './styles/GLobal.styled';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+    <Router>
+      <GlobalStyle/>
+    <Main/>
```

Comment:
Ideally index.js contains the logic for rendering your App component and provides it with necessary props like Context, GlobalStyles etc. You should render App component here instead of Main and render other components inside App.